committeeoversightapp/templates/committeeoversightapp/category_detail_page.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div class="col-12 col-lg-9 mx-auto">
+    <h1 class="mb-4">Category {{page.category.id}}: {{page.category.name}}</h1>
+  </div>
+
+  {% include 'partials/streamfield.html'%}
+
+  {% include 'partials/hearing_table.html' %}
```

Comment:
this is rendering funny for me.

![Screen Shot 2019-09-11 at 3 36 09 PM](https://user-images.githubusercontent.com/12176173/64733238-f961ab80-d4a9-11e9-8a05-558995288ca3.png)


i think your top-level element in the stream field partial may need to be a col element, but i'm not sure!

Reply:
This PR is for getting all the right data on the page, not the design! There's lots to be done there.